client/state/data-layer/wpcom/account-recovery/index.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Internal dependencies
+ */
+import wpcom from 'lib/wp';
+
+import {
+	ACCOUNT_RECOVERY_RESET_OPTIONS_RECEIVE,
+	ACCOUNT_RECOVERY_RESET_OPTIONS_ERROR,
+} from 'state/action-types';
+
+export const requestResetOptions = ( { dispatch }, { userData } ) => (
+	wpcom.undocumented().accountRecoveryReset( userData ).getResetOptions()
+		.then( options => dispatch( {
+			type: ACCOUNT_RECOVERY_RESET_OPTIONS_RECEIVE,
+			options,
+		} ) )
+		.catch( error => dispatch( {
+			type: ACCOUNT_RECOVERY_RESET_OPTIONS_ERROR,
+			error,
+		} ) )
+);
```

Comment:
Reasonable. Moved and updated by 6da676f80799ed232d52d05461635f9c05b793a6 and 7d1b9f1bf67182f7bb73f14953b0492b89c448e8.